### PR TITLE
Sanity powered blog ✈️

### PR DESCRIPTION
**Pull request by joshuawootonn**

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
+const withMakeswiftInitializer = require('@makeswift/runtime/next/plugin')
 
-const withMakeswift = withMakeswiftInitializer();
+const withMakeswift = withMakeswiftInitializer()
 
 module.exports = withMakeswift({
   reactStrictMode: true,
   images: {
-    domains: ["s.mkswft.com"],
+    domains: ['s.mkswft.com', 'cdn.sanity.io'],
```
Comment (migueloller) [inline next.config.js]: You shouldn't have to provide `s.mkswft.com` as it should be included by default with the Makeswift Next.js plugin

File: tailwind.config.js [hunk 2]
```diff
@@ -39,6 +39,11 @@ module.exports = {
         '7xl': ['4.5rem', { lineHeight: '1' }],
         '8xl': ['6rem', { lineHeight: '1' }],
         '9xl': ['8rem', { lineHeight: '1' }],
+        // straight from MakeswiftTypographies
```
Comment (migueloller) [inline tailwind.config.js]: What is "MakeswiftTypographies"?

**Review by migueloller — APPROVED** [review]
This looks awesome!

File: lib/config.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+ */
+export function getPublicConfig(): PublicConfig {
+  invariant(
+    process.env.NEXT_PUBLIC_SANITY_DATASET,
+    'NEXT_PUBLIC_SANITY_DATASET env var is not defined.',
+  )
+  invariant(
+    process.env.NEXT_PUBLIC_SANITY_PROJECT_ID,
+    'NEXT_PUBLIC_SANITY_PROJECT_ID env var is not defined.',
+  )
```
Comment (flybayer) [inline lib/config.ts:31]: what do I need to set these two env vars to?

File: tailwind.config.js (hunk 2 above)
Comment (joshuawootonn) [reply tailwind.config.js]: Good catch. It's nothing official was just trying to reference the typographies that are created within the Makeswift site.

File: lib/config.ts (hunk 3 above)
Comment (joshuawootonn) [reply lib/config.ts:31]:
```
NEXT_PUBLIC_SANITY_PROJECT_ID=zg35kxbb
NEXT_PUBLIC_SANITY_DATASET=production
```

Comment (joshuawootonn) [reply lib/config.ts:31]: CleanShot 2022-10-17 at 11 33 10@2x
This is where the values are coming from

File: tailwind.config.js (hunk 2 above)
Comment (migueloller) [reply tailwind.config.js]: Ah, ok got it!

**Comment by joshuawootonn** [comment]
I merged this branch, and it caused some issues in the Makeswift builder. Flightcontrol.dev was fine, but I didn't want to block anyone, so I reverted the branch till I have time to try again tomorrow.

**Pull request merged**